docs/non-normative/k8s-naming.md

```diff
@@ -0,0 +1,23 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: K8s naming
+--->
+
+# Kubernetes Naming Guidance
```

Comment:
could you please add a link to this doc with 1-liner summary (like "k8s conventions ) to the https://github.com/open-telemetry/semantic-conventions/blob/9cc9f96a588fc1b6644aae957fb4a73fd23e10c9/docs/general/naming.md?plain=1#L74-L76?

Something along the lines of:

> Known exceptions include [K8s API names](todo link) where single word is used for consistency with the instrumented API.

Reply:
added